project/build.properties

```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-sbt.version=0.13.18
+sbt.version=1.3.13
```

Comment:
1.4.0 is unstable?

Reply:
Looks like it came out 3 days ago. It could be totally fine; if there's any question we can address it later, too.

Reply:
Upgrade from 1.3.13 to 1.4 will require to remove compiler reporter snippet (pretty ancient one), otherwise build import fails on compilation error (sbt internals changed in related to this snippet classes). See this comment on why I skipped it in this PR - https://github.com/apache/spark/pull/29286#issuecomment-704546933.

Just to give an impression - this is a starting point to unblock move to 1.4.0 https://github.com/apache/spark/commit/1e14d4aa8a38c11c3e46ea85c668b27d3106ad2c. After that one would need to fix a lot of warnings that are missed right now.